starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +23,37 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (words){
+  for ( i = 0; i < words.length; i ++){
+    lengthNumber += words[i].length;
+    return Math.max(lengthNumber)
+  }
```

Comment:
For this exercise you should:
- create a variable `longestWord` for saving the longest word
- on each for iteration, compare the current word's length and change if longest
- after the `for`, return `longestWord`